### PR TITLE
DRS-484 drs hacky

### PR DESCRIPTION
**Pull request by ddietterich**
- Implemented basic DRS API
- Revised services to support it
- Added integration test fixtures for DRS
- Added DrsTest integration test

### CODE REVIEW

File: src/main/java/bio/terra/controller/DataRepositoryServiceApiController.java [hunk 1]
```diff
@@ -74,6 +75,12 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
         return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler
+    public ResponseEntity<DRSError> notImplementedExceptionHandler(NotImplementedException ex) {
+        DRSError error = new DRSError().msg(ex.getMessage()).statusCode(HttpStatus.NOT_FOUND.value());
```
Comment (thathert) [inline src/main/java/bio/terra/controller/DataRepositoryServiceApiController.java]: Is this not supposed to be `NOT_IMPLEMENTED` here as well?

File: src/main/resources/data-repository-openapi.yaml [hunk 2]
```diff
@@ -1094,6 +1070,22 @@ paths:
...
+          type: boolean
+          default: false
+          description: >-
+            If false and the object_id refers to a bundle, then the ContentsObject array
+            contains only those objects directly contained in the bundle. That is, if the
+            bundle contains other bundles, those other bundles are not recursively
+            included in the result.
+            
+            If true and the object_id refers to a bundle, then the entire set of objects
+            in the bundle is expanded. That is, if the bundle contains aother bundles,
```
Comment (thathert) [inline src/main/resources/data-repository-openapi.yaml]: aother -> other

File: src/test/java/bio/terra/integration/DrsTest.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest // TODO: Do we need this?
```
Comment (thathert) [inline src/test/java/bio/terra/integration/DrsTest.java]: I don't think we are using any of the [features this enables](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/test/context/SpringBootTest.html).

**Review by thathert — APPROVED** [review]
LGTM, a few tiny things. Once we finish with filesystem cloning and the directory work we can revisit this.

File: src/main/java/bio/terra/controller/DataRepositoryServiceApiController.java (hunk 1 above)
Comment (ddietterich) [reply src/main/java/bio/terra/controller/DataRepositoryServiceApiController.java]: Er... yes. Thanks for catching that!

**Pull request merged**